.github/workflows/kind_integration.yml

```
@@ -65,6 +68,39 @@ jobs:
         name: image-archives
         path: /home/runner/archives
   # todo: Keep in sync with `release.yml`
+  windows_static_cli_tests:
+    name: Static CLI tests (windows)
+    runs-on: windows-latest
+    needs: [docker_build]
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Try to load cached Go modules
+      # actions/cache@v1.1.2
+      uses: actions/cache@70655ec
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Download image archives
+      # actions/download-artifact@v1
+      uses: actions/download-artifact@18f0f59
+      with:
+        name: image-archives
+    - name: Install linkerd CLI
+      shell: bash
+      run: |
+        echo ::set-env name=TAG::$(CI_FORCE_CLEAN=1 bin/root-tag)
+        echo $TAG
+        image="gcr.io/linkerd-io/cli-bin:main"
```

Have that other docker issue going on, but will want to use `$TAG` here!

```suggestion
        image="gcr.io/linkerd-io/cli-bin:$TAG"
```

---

yeah, totally. Just want to check if these commands actually work! wanted to switch to $TAG once I see things work.